### PR TITLE
docs: add copyright holders to license

### PR DESCRIPTION
**Pull request by lostmygithubaccount**
noticed this in the ibis-birdbrain repo, feel free to change directly or suggest other words

### CODE REVIEW

**Comment by lostmygithubaccount** [comment]
unrelated docker failure that was noticed elsewhere?

File: LICENSE.txt [hunk 1]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright Ibis developers
```
Comment (gforsyth) [inline LICENSE.txt:190]: Needs a date to be valid

**Review by gforsyth — CHANGES_REQUESTED** [review]
docker should fix itself when it re-runs

**Comment by gforsyth** [comment]
Merging this in -- don't need CI to run on a license file.

**Comment by gforsyth** [comment]
Wait, I lied, I _do_ need CI to run :shrug:

**Pull request merged**